src/main/java/com/passpnu/passwordmanager/controller/UserController.java

```diff
@@ -0,0 +1,46 @@
+package com.passpnu.passwordmanager.controller;
+
+import com.passpnu.passwordmanager.dto.UserDto;
+import com.passpnu.passwordmanager.service.UserEntityServiceImpl;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@AllArgsConstructor
+public class UserController {
+    private final UserEntityServiceImpl userEntityService;
+    private final AuthenticationManager authenticationManager;
```

Comment:
You should always prefer autowiring interfaces over concrete implementations (https://stackoverflow.com/questions/66794100/autowiring-interface-vs-class)

Reply:
Fixed